packages/hash/frontend/src/theme/components/inputs/MuiOutlinedInputThemeOptions.ts

```diff
@@ -0,0 +1,48 @@
+import { Components, Theme } from "@mui/material";
+
+export const MuiOutlinedInputThemeOptions: Components<Theme>["MuiOutlinedInput"] =
+  {
+    defaultProps: {
+      notched: false,
+    },
+    styleOverrides: {
+      root: ({ theme }) => ({
+        color: "inherit",
+        lineHeight: "18px",
+        borderRadius: "6px",
+        "&:hover": {
+          "& .MuiOutlinedInput-notchedOutline": {
+            borderColor: theme.palette.gray[30],
+          },
+        },
```

Comment:
Currently I'm seeing no hover state, so just want to clarify whether this addition was intentional?

Reply:
The default hover state is a different color, so this helps ensure that it doesn't change colors.

I also don't see a different hover state for hover fields on the Figma. If this is something we want to clarify further with Maggie, I would suggest taking it up as a follow-up and not as a part of this PR.